deeppavlov/configs/classifiers/superglue/superglue_record_roberta.json

```diff
@@ -1,72 +1,32 @@
 {
-  "metadata": {
-    "variables": {
-      "BASE_MODEL": "roberta-large",
-      "TASK": "record",
-      "ROOT_PATH": "~/.deeppavlov",
-      "DOWNLOADS_PATH": "{ROOT_PATH}/downloads",
-      "MODELS_PATH": "{ROOT_PATH}/models",
-      "MODEL_PATH": "{MODELS_PATH}/classifiers/superglue_record_{BASE_MODEL}",
-      "BINARY_CLASSIFICATION": false,
-      "SEED": 42
-    },
-    "download": [
-      {
-        "url": "http://files.deeppavlov.ai/0.17/classifiers/superglue/superglue_record_roberta.tar.gz",
-        "subdir": "{MODELS_PATH}"
-      }
-    ]
-  },
   "dataset_reader": {
     "class_name": "huggingface_dataset_reader",
-    "path": "super_glue",
+    "path": "{COMPETITION}",
     "name": "{TASK}",
     "train": "train",
     "valid": "validation",
-    "seed": "{SEED}",
-    "downsample_ratio": [
-      1.8,
-      1.8
-    ],
+    "test": "test",
+    "downsample_ratio": [1.8, 1.8, 1.8],
```

Comment:
Was [1.8, 1.8]. Additional value is typo or intentional?

Reply:
Without the additional 1.8 (probably, better to change to 1) - the confing won’t work, since I added a test split (so now there are 3 splits: train, valid, test)